src/app/mandate/mandate.component.html

```diff
@@ -0,0 +1,146 @@
+          receipt.</p>
+        <p>If you have set a donor account password you may also
+          view your <a href="/my-account/donations">donation history</a>.
+        </p>
+      }
+      @if (false) {
+        <p class="error" aria-live="assertive">
+          Your donation was not processed immediately &ndash; it may be that there is a short delay because we're very
+          busy. Please refresh this page to see the latest status. We'll also email your receipt within a few minutes when
+          your donation is processed.
+        </p>
+      }
+      <div aria-live="polite">
+        @if (complete) {
+          <div>
+            <div class="screenreader-only">
+              Thank you! Your generous regular donation has been set up.
+            </div>
+          </div>
+        }
+      </div>
+      @if (complete) {
+        <div>
+          <div class="thank-you">
+            <h2>Thank you!</h2>
+            <p class="thank-you-text">Your generous regular donation has been set up.</p>
+          </div>
+          <hr aria-hidden="true">
+          <div class="donation-summary">
+<!--            <p><strong>Hi {{ donation.firstName }},</strong></p>-->
+            <p><strong>Your donation summary</strong></p>
+            <p>Your donation of
+              <strong class="figure">{{ mandate.amount.amountInPence + ' ' + mandate.amount.currency }}</strong>
+              is worth
+              <strong class="figure">{{ totalValue + ' ' + mandate.amount.currency }}
+              each month!</strong>
+            </p>
+            <br/>
+<!--            @if (mandate?.thankYouMessage) {-->
+<!--              <p id="charity-thank-you">{{ mandate?.thankYouMessage }}</p>-->
+<!--            }-->
+          </div>
+          <div>
+            <div class="receipt">
+              <p><strong>Your Regular Donation Schedule</strong></p>
+              <!-- Date time appears as e.g. "Sep 5, 2023, 2:54 PM" -->
+              <table>
+                <tr>
+                  <td>Active from</td>
+                  <td>{{ mandate.schedule.activeFrom | date: 'shortTime'}} {{ mandate.schedule.activeFrom | date: 'mediumDate'}}</td>
+                </tr>
+                <tr>
+                  <td>Day of month</td>
+                  <td>{{ mandate.schedule.dayOfMonth | date: 'shortTime'}} {{ mandate.schedule.dayOfMonth | date: 'mediumDate'}}</td>
+                </tr>
+                <td></td>
+                <tr class="donationAmount">
+                  <td>Your donation</td>
+                  <td>{{ mandate.amount.amountInPence }}  {{ mandate.amount.currency }}</td>
+                </tr>
+<!--                @if (donation.matchedAmount > 0) {-->
+                  <tr>
+                    <td>Matched Amount</td>
+                    <td>{{ mandate.amount.amountInPence }} {{ mandate.amount.currency }}</td>
+                  </tr>
```

Comment:
this isn't called matched amount in the mandate object, I don't think we should present it as matched amount here for same reason as above - its only going to be matched for some of the donations not all.

Actually it might be more accurate to say the donation amount is 10 GBP per month, and the matched amount is 30 GBP. But that matched amount figure should come from the mandate object. Fine to add another property to the mandate here and we'll put it in the matchbot api response later.

Reply:
Thank you @bdsl , I think my last commit should resolve all your suggestions!

Reply:
I don't think it does - this table row still has a heading that doesn't match what's displayed. If the heading is "Matched Amount" then IMHO we should be displaying a field in the mandate called "matchedAmount". It might be the same number as `amount` but if so I think that's something that should be programmed into matchbot, not frontend.

I think we should aim for making it so that as a user if you happen to know JSON you learn the same things by either reading this page as normal or directly reading the Mandate as a JSON object from matchbot - frontend shouldn't be doing business logic to add extra information.


We could leave the matched amount off this page for now and have that as another todo on the ticket or a new ticket - not sure. Or add it as another field on the mandate object as part of this.

Reply:
thank you @bdsl , yes now updated!